cmd/gcContent/gcContent.go

```diff
@@ -0,0 +1,96 @@
+// Command Group: "FASTA and Multi-FASTA Tools"
+
+// Calculates the gc content of fasta sequences for all regions specified in a bed file
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/dna"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+)
+
+func gcContent(bedFile string, faFile string, outFile string, multiFaMode bool, species string) {
+
+	var currRecordSeq []dna.Base
+	var found bool
+	var gc float64
+	var annotated bed.Bed
+	var speciesStart, speciesEnd int
+
+	regionsChan := bed.GoReadToChan(bedFile)
+	records := fasta.Read(faFile)
+	recordsMap := fasta.ToMap(records)
+	out := fileio.EasyCreate(outFile)
+
+	for curr := range regionsChan {
+
+		if multiFaMode {
+			// when multiFa, each fasta sequence is a species, e.g. >human, >hca, assuming that the user specified the correct chromosome, e.g. this multi-fasta is chr1.fa
+			currRecordSeq, found = recordsMap[species]
+			if found {
```

Comment:
Stylistic point, but I find it is a bit more aesthetically pleasing if you do the negative check first with a log fatal, so your main code block doesn't have to have an extra indent. ie: 

if !found {
                log.Fatal
}

rest of the code here

Reply:
Done!